crates/katana/rpc/rpc/src/lib.rs

```diff
@@ -36,6 +37,9 @@ pub async fn spawn(sequencer: Arc<KatanaSequencer>, config: ServerConfig) -> Res
             ApiKind::Katana => {
                 methods.merge(KatanaApi::new(sequencer.clone()).into_rpc())?;
             }
+            ApiKind::Dev => {
+                methods.merge(KatanaApi::new(sequencer.clone()).into_rpc())?;
```

Comment:
```suggestion
                methods.merge(DevApi::new(sequencer.clone()).into_rpc())?;
```

Supposed to be the new `DevApiServer` implementation